src/GraphQL/Execution/QueryMissingError.cs

```diff
@@ -0,0 +1,39 @@
+namespace GraphQL.Execution;
+/// <summary>
+/// Represents an error indicating that no GraphQL query was provided to the request.
+/// </summary>
+public class QueryMissingError : RequestError
+{
+    /// <inheritdoc cref="QueryMissingError"/>
+    public QueryMissingError() : base("GraphQL query is missing.") { }
+}
+
+/// <summary>
+/// Represents an error that occurred prior to the execution of the request.
```

Comment:
```suggestion
/// Represents an error that occurred prior to the execution of the GraphQL request.
/// This refers to any errors that arise before passing the request inside the GraphQL engine, that is, even before its validation.
```

Reply:
And please move into separate file in `Error` folder.

Reply:
👍